pulsar/consumer_partition.go

```diff
@@ -1775,14 +1775,6 @@ func (pc *partitionConsumer) internalClose(req *closeRequest) {
 		return
 	}
 
-	if state == consumerClosed || state == consumerClosing {
```

Comment:
Could we move this state check before line 1770? It would be useful to log the error if the consumer is already closed.

Reply:
Moved.